pkg/controller/storagecluster/reconcile.go

```diff
@@ -56,6 +56,10 @@ osd_memory_target_cgroup_limit_ratio = 0.5
 	monCountOverrideEnvVar = "MON_COUNT_OVERRIDE"
 	// EBS represents AWS EBS provisioner for StorageClass
 	EBS StorageClassProvisionerType = "kubernetes.io/aws-ebs"
+	// If the CleanupPolicyLabel is set to the delete policy, ocs-operator will
+	// pass the information to rook to destroy the data.
+	CleanupPolicyLabel  = "cleanup.ocs.openshift.io"
+	CleanupPolicyDelete = "yes-really-destroy-data"
```

Comment:
I don't think it's needed if you use the type

Reply:
Same as above.